sci-libs/ccfits/ccfits-2.5.recipe

```diff
@@ -0,0 +1,114 @@
+SUMMARY="C++ Wrappers for the cfitsio library"
+DESCRIPTION="CCfits is an object oriented interface to the cfitsio library. \
+It is designed to make the capabilities of cfitsio available to programmers \
+working in C++. It is written in ANSI C++ and implemented \
+using the C++ Standard Library with namespaces, exception handling, \
+and member template functions."
+HOMEPAGE="https://heasarc.gsfc.nasa.gov/fitsio/CCfits"
+COPYRIGHT="2002-2019 Ben Dorman, Craig Gordon, Bryan Irby - HEASARC, NASA/GSFC"
+LICENSE="CCFITS"
+REVISION="1"
+SOURCE_URI="https://heasarc.gsfc.nasa.gov/fitsio/CCfits/CCfits-$portVersion.tar.gz"
+CHECKSUM_SHA256="938ecd25239e65f519b8d2b50702416edc723de5f0a5387cceea8c4004a44740"
+SOURCE_DIR="CCfits"
+PATCHES="ccfits-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86_64 ?arm ?ppc ?sparc"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="0.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	ccfits$secondaryArchSuffix = $portVersion compat >= 2.5
```

Comment:
Why compat >= 2.5 ?

Reply:
They used to have a messed up versioning schema. Future versions SHOULD be compat with 2.5. But ok, I'll remove it